### ISSUE TITLE
md-data-table eating drop events

### ISSUE THREAD

**Issue by f1tech** [daniel-nagy/md-data-table#276]
I'm trying to make a row or a cell within a row a drop target. On the table row, the table cell and a span within the cell I've tried adding:

ondrop="order.dropFromWeb(event)"  ondragover="order.allowDrop(event)" ondragexit="order.allowDrop(event)" ondragenter="order.allowDrop(event)"

All of these are removed from the HTML when it's rendered. I know I should probably create a directive, but was trying to get something up and running first.

**Comment by daniel-nagy** [comment]
I wrote a sortable directive a few months ago that would reorder an arbitrary list of elements, including table rows, by drag and drop. I did not use HTML5's drag and drop API because I remember it being a problem with table rows.

My data table module should not remove attributes, I suspect that the browser is doing this because HTML5 drag and drop attributes within a table are invalid?

I've been given permission to open source some of the directives/modules I've written for the people I work for. I may open source my sortable directive soon.
